html/global_attributes.json

```
@@ -1023,6 +1036,7 @@
       "inputmode": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Global_attributes/inputmode",
+          "spec_url": "https://html.spec.whatwg.org/multipage/interaction.html#input-modalities:-the-inputmode-attribute",
```

Alternatively (consistently), #attr-inputmode but what you have works, too.

```suggestion
          "spec_url": "https://html.spec.whatwg.org/multipage/interaction.html#attr-inputmode",
```